google/cloud/storage/examples/run_examples_production.sh

```diff
@@ -19,7 +19,7 @@ set -eu
 # This script should is called from the build directory, and it finds other
 # scripts in the source directory using its own path.
 readonly BINDIR="$(dirname $0)"
-source "${BINDIR}/../../ci/colors.sh"
+source "${BINDIR}/../../../../ci/colors.sh"
```

Comment:
Should this be something like root-level or project-level?

Reply:
That would be nice, but we do not have a global variable like `PROJECT_ROOT`.  Sometimes I define one to avoid repetition, but here it would be used once.  Created #679 to fix these.